blog/2025-02-21/index.md

```diff
@@ -24,36 +24,72 @@ From a technical perspective, Flatpak does not require elevated privileges to in
 
 Flatpak apps are also built from a declarative manifest, which defines the exact sources and environment to build from to enable auditability and as much reproducibility as possible.
 
-Due to Flatpak’s sandboxing, apps don’t have permission to access to many aspects of the host OS or user data they might need. To get that access, apps must request it using Portals or static permissions.
+Due to Flatpak’s sandboxing, apps don’t have permission to access many aspects of the host OS or user data they might need. To get that access, apps must either request it using Portals or static permissions.
```

Comment:
Not a native speaker, but in my head, your not requesting static permissions

```suggestion
Due to Flatpak’s sandboxing, apps don’t have permission to access many aspects of the host OS or user data they might need. To get that access, apps must either request it using Portals or use static permissions.
```

Reply:
added